scripts/rook.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -e
+
+ROOK_VERSION=${ROOK_VERSION:-"v1.1.7"}
+ROOK_DEPLOY_TIMEOUT=${ROOK_DEPLOY_TIMEOUT:-300}
+
+function deploy_rook() {
+    ROOK_URL=https://raw.githubusercontent.com/rook/rook/${ROOK_VERSION}/cluster/examples/kubernetes/ceph
```

Comment:
define `ROOK_URL` above and use it in  both deploy_rook and celan_rook ( i prefer to call it as  teadown_rook)

Reply:
> define ROOK_URL above and use it in both deploy_rook and celan_rook ( i prefer to call it as teadown_rook)

nice catch, I fix that ROOK_URL is used by both. and yes, teardown is look clear than clean. I will fix it.